functions/firestore/ItemHelper.js

```diff
@@ -15,9 +15,12 @@ const createItem = body => new Promise(async (resolve, reject) => {
 
 const modifyItem = body => new Promise(async (resolve, reject) => {
   try {
-    console.log(`Updating item '${body.itemId}'...`);
+    console.log(`Updating item '${(body.itemId)}'...`);
+    const oldState = (await REFS.COLLECTIONS.ITEMS.doc(body.itemId).get()).data();
```

Comment:
ha nem letezik az oldState  azt le kene majd  kezelni, ne crasheljen ezen

Reply:
Azon a sima update is crashelt volna, nem?

Reply:
jap,  ezert kezeltem le a modifyUser-ben. de igazabol most lenyegtelen a holnapi progresshez

Reply:
akkor odarakok egy TODO kommentet, aztan mehet a merge?

Reply:
ah, megcsinalom rendesen

Reply:
de persze, siman mehet :D

Reply:
Done.